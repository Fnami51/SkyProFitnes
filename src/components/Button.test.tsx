import React from 'react' // нужно для тестов
import { render } from '@testing-library/react'
import Button from './Button'

test('Button matches snapshot', () => {
	const { asFragment } = render(
		<Button variant='primary' onClick={() => {}}>
			Click me
		</Button>
	)
	expect(asFragment()).toMatchSnapshot()
})
