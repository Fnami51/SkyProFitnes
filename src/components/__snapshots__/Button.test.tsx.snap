// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches snapshot 1`] = `
<DocumentFragment>
  <button
    class="font-roboto h-[52px] font-normal text-lg leading-[110%] text-center rounded-[46px] transition-all duration-300 ease-in-out cursor-pointer bg-primary text-black hover:bg-primary-hover active:bg-black active:text-white "
    type="button"
  >
    Click me
  </button>
</DocumentFragment>
`;
